eclipse-collections-code-generator/src/main/resources/impl/map/mutable/objectPrimitiveHashMapWithHashingStrategy.stg

```diff
@@ -535,6 +535,20 @@ public class Object<name>HashMapWithHashingStrategy\<K> implements MutableObject
         return value;
     }
 
+    @Override
+    public <type> getAndPut(K key, <type> putValue, <type> defaultValue)
+    {
+        int index = this.probe(key);
+        if (isNonSentinel(this.keys[index]) && nullSafeEquals(this.toNonSentinel(this.keys[index]), key))
+        {
+            <type> existingValue = this.values[index];
+            this.addKeyValueAtIndex(key, putValue, index);
```

Comment:
Same question as above, can this just be `this.values[index] = putValue;`?

Reply:
Fixed.